authentication.ts

```diff
@@ -0,0 +1,65 @@
+import * as express from "express";
+import { createRemoteJWKSet, jwtVerify } from "jose";
+import { JwtError } from "@controllers/public-rest";
+
+// Create a JWKS client using Keycloak's JWKS endpoint
+const JWKS = createRemoteJWKSet(new URL(process.env.KEYCLOAK_CERTS!));
```

Comment:
We have all env var access in the config.ts file.
So best to add 
`export const KEYCLOAK_CERTS = process.env.KEYCLOAK_CERTS;` there like the rest.

Reply:
When we'll adding real secrets, we'll move to a .env file, right?

Reply:
It still comes from .env, it's just that everything from env and other config stuff is accessed through the config.ts file to keep a better overview of the parameters that go into the application.